src/page-components/projects/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Project } from '../../../plugins/gatsby-source-cd-airtable/src/interfaces/project'
+import { Layout, Section, SectionContent } from 'components/layout'
+import { useTranslation } from 'gatsby-plugin-react-i18next'
+import { HighlightedProject } from './sections'
+
+interface ProjectsPageProps {
+  data: {
+    allProject: {
+      edges: {
+        node: Project
+      }[]
+    }
+  }
+}
+
+const ProjectsPage: React.FC<ProjectsPageProps> = ({
+  data,
+}: ProjectsPageProps) => {
+  const { t } = useTranslation()
+  const highlightedProject = React.useMemo(() => data.allProject.edges[0], [
+    data,
+  ])
+
+  const restProjects = React.useMemo(() => data.allProject.edges.slice(1), [
+    data,
+  ])
+
+  return (
+    <Layout crumbs={[{ label: t('pages.projects.navigation.projects') }]}>
+      <Section>
+        <SectionContent>
+          <h1>{t('pages.projects.title')}</h1>
+          <p>{t('pages.projects.description')}</p>
+          <ul>
+            {restProjects.map((project) => (
+              <li key={project.node.originalId}>
+                <ul data-cy="project">
+                  <li data-cy="project__name">{project.node.name}</li>
+                  {project.node.tagline && (
+                    <li data-cy="project__tagline">{project.node.tagline}</li>
+                  )}
+                </ul>
+              </li>
+            ))}
+          </ul>
+        </SectionContent>
+      </Section>
+      <Section>
+        <SectionContent>
+          {/* FIXME: provide real data from airtable */}
```

Comment:
@HormCodes muzu nejak dostat do Airtablu a rozšířit interface?

Reply:
Přístup do AirTable dám. Pošli mi prosím na Slacku v DM mail. Existuje tam už tabulka s projekty. Klidně bych všechny smazal a vložil třeba tři jako demo, kde model bude vypadat: 

- csName
- enName
- csTagline
- enTagline
- csSlug
- enSlug
- coverUrl
- iconUrl
- csDescription
- enDescription
- githubUrl
- slackChannelUrl
- slackChannelName
- trelloUrl
- wikiUrl
- projectUrl
- tags
- volunteers
- progress (0-100)
- projectLead
- ... snad zatím vše

Už jsem trochu začal a asi sám vystavím PR na úpravu datového pluginu.